UPGRADE-FROM-1.12-WITH-PRICE-HISTORY-PLUGIN-TO-1.13-LEGACY.md

```diff
@@ -0,0 +1,131 @@
+⚙️ Upgrade from Sylius 1.12 with PriceHistoryPlugin to Sylius 1.13
+==================================================================
+
+We encourage you to use the upgrade instructions based on Rector as it is more convenient and faster to accomplish.
+The rector installation guide is available [here](UPGRADE-FROM-1.12-WITH-PRICE-HISTORY-PLUGIN-TO-1.13.md).
+
+Legacy upgrade
+--------------
+
+1. Remove the PriceHistoryPlugin from composer.json by running:
+
+    ```bash
+    composer remove sylius/price-history-plugin --no-scripts
+    ```
+
+1. Make sure to remove the following config from your `config/packages/sylius_price_history_plugin.yaml` file:
+
+    ```diff
+    -  - { resource: "@SyliusPriceHistoryPlugin/config/config.yaml" }
+    ```
+   And the route from your `config/routes/sylius_price_history_plugin.yaml`:
+    ```diff
+    - sylius_price_history_admin:
+    -    resource: '@SyliusPriceHistoryPlugin/config/admin_routing.yaml'
+    -    prefix: '/%sylius_admin.path_name%'
+    ```
+   And also remove this line from your `packages/bundles.php` file:
+    ```diff
+    - Sylius\PriceHistoryPlugin\SyliusPriceHistoryPlugin::class => ['all' => true],
+    ```
+
+1. Update the `Channel` entity to use interface from `Core`:
+
+    ```diff
+    - use Sylius\PriceHistoryPlugin\Domain\Model\ChannelInterface;
+    + use Sylius\Component\Core\Model\ChannelInterface;
+    ```
+   Then remove the trait:
+    ```diff
+    - use ChannelPriceHistoryConfigAwareTrait;
+    ```
+
+1. Update the `ChannelPricing` entity to use interface from `Core`:
+
+    ```diff
+    - use Sylius\PriceHistoryPlugin\Domain\Model\ChannelPricingInterface;
+    + use Sylius\Component\Core\Model\ChannelPricingInterface;
+    ```
+   Then remove the trait:
+    ```diff
+    - use LowestPriceBeforeDiscountAwareTrait;
+    ```
+
+1. Update your resources configuration for `ChannelPricingLogEntry` and `ChannelPriceHistoryConfig` if you have changed them in your project:
+
+    ```diff
+    - sylius_price_history:
+    + sylius_core:
+         resources:
+             channel_price_history_config:
+                 classes:
+                     model: App\Entity\ChannelPriceHistoryConfig
+                     ...
+             channel_pricing_log_entry:
+                 classes:
+                     model: App\Entity\ChannelPricingLogEntry
+                     ...
+    ```
+
+   followed by this change to configure the `batch_size` option in the upgraded version, you will need to use the `sylius_core.price_history.batch_size` root node instead of `sylius_price_history.batch_size`.
```

Comment:
Would be nice to have this change either spread out like the one above or incorporated within it